Formula/pyqt5.rb

```diff
@@ -15,8 +16,8 @@ class Pyqt5 < Formula
 
   deprecated_option "enable-debug" => "with-debug"
 
-  depends_on :python3 => :recommended
-  depends_on :python => :optional
+  depends_on :python => :recommended
+  depends_on :python3 => :optional
```

Comment:
We pretty much always mandate our own Python 3 so we've not had complaints about that.

I think it's worth just leaving this file as-is for now (or at least pulling into another PR)